src/main/java/com/aws/greengrass/clientdevices/auth/iot/IotAuthClient.java

```diff
@@ -170,12 +169,6 @@ public Stream<Thing> getThingsAssociatedWithCoreDevice() {
                 return response.associatedClientDevices().stream()
                         .map(AssociatedClientDevice::thingName)
                         .map(Thing::of);
-            } catch (Exception e) {
```

Comment:
This got removed because otherwise using the RetryUtils doesn't work very well given we don't just simply want to retry on `CloudServiceInteractionException` we want the caller to have more control over what to do with the exceptions